src/Raven.Studio/typescript/viewmodels/common/detectBrowser.ts

```diff
@@ -0,0 +1,43 @@
+import viewModelBase = require("viewmodels/viewModelBase");
+import studioSettings = require("common/settings/studioSettings");
+
+class detectBrowser extends viewModelBase {
+
+    showBrowserAlert = ko.observable<boolean>(false);
+    dontShowBrowserAlertAgain = ko.observable<boolean>(false);
+    
+    constructor(private settingsHasInfo: boolean) {
```

Comment:
can we call that: rememberSettings or persistChoice, etc?

Reply:
How about: `infoPersistedInSettings `